adhocteam/Head-Start-TTADP
# Context menu can open up instead of always down

## Description of change

There are CSS issues overflowing in one axis while scrolling in a different axis. If one axis scrolls the other axis is set to auto. The result is context menus get cut off if at the bottom of a scrolling element. This situation happens with the activity reports table. Added the ability for the context menu to open up. The last element of the activity report list now has a usable context menu.

## How to test

1) Pull down changes
2) Have one report in the reports list (may have to approve a report)
3) Open the context menu of the last item in the reports list, the menu should open up and both options select-able

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-117

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
